psydac/api/fem.py

```diff
@@ -372,42 +440,89 @@ def get_space_indices_from_target(self, domain, target):
                 j = i
         return i,j
 
-    def construct_arguments(self, backend=None):
+    def construct_arguments(self, backend=None, with_openmp=False):
```

Comment:
Please add docstring.

Also note that this function should not change `self`, but only return some arguments.

Reply:
done in [ec10ae7](https://github.com/pyccel/psydac/pull/213/commits/ec10ae70e2106c6cc339a5aa2ae5ac81c351a7f6) and [faee8ae](https://github.com/pyccel/psydac/pull/213/commits/faee8aee6c7650f331121b939f604925c9dd86db)